atd-vze/src/views/Crashes/personDataMap.js

```diff
@@ -10,23 +10,26 @@ const shouldRenderVictimName = (data, roles) => {
 
 const getInjurySeverityColor = personRecord => {
   switch (personRecord["prsn_injry_sev_id"]) {
-    case 0: // UNKNOWN
+    case 0: // UNKNOWN: no color
       return "muted";
-    case 5: // NOT INJURED
-      return "primary";
-    case 1: // SUSPECTED INCAPACITATING INJURY
-      return "warning";
-    case 2: // SUSPECTED MINOR INJURY
+    case 1: // SUSPECTED INCAPACITATING INJURY: yellow
       return "warning";
-    case 3: // POSSIBLE INJURY
+    case 2: // SUSPECTED MINOR INJURY: yellow
       return "warning";
-    case 4: // FATAL INJURY
+    case 3: // POSSIBLE INJURY: blue
+      return "primary";
+    case 4: // FATAL INJURY: red
       return "danger";
-    default:
-      break;
+    case 99: // KILLED: red
+      return "secondary"
+    default: // Other cases: Not reported, Reported invalid, Not injured
+      return "secondary";
   }
 };
 
+console.log(getInjurySeverityColor({ prsn_injry_sev_id: 1 }));
```

Comment:
We probably haven't touched on this a lot as a team, but we usually remove console.logs that were used for debugging. My personal approach is that I'll remove it if it isn't going to help the next person in the code or surface something really important. Not a big deal at all but wanted to give some context.

This one seems like it can go, but feel free to push back! 🙏

Reply:
Just made the change! Will also remember this for the future :)